### PR TITLE
Add tour kit component

### LINKED ISSUE #32892: Implement the new spotlight component
https://github.com/woocommerce/woocommerce/issues/32318#issuecomment-1104960269

Following up on the [new spotlight component initiative](https://ghidorahp2.wordpress.com/2022/04/13/shared-in-app-spotlight-component/), please implement the new spotlight that’s part of a larger design element used for help and guidance.

The design will be the same proposed by Dotcom with a few tweaks to make it consistent with WooCommerce styles and usage.

image

**Design overview:** 
cJQeyyhx9PraQrY3YAtCGM-fi-888%3A112694

**Differences:**
- Use WooCommerce button styles and colors.
- Remove the Minimize icon.
- Use SF Pro Text for the title in place of Inter.

**Updated shipping smart defaults designs (with the new spotlight component)**: 
cJQeyyhx9PraQrY3YAtCGM-fi-705%3A80783

### PR DESCRIPTION
**Pull request by chihsuan**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #32892.

Add woo tour kit component

Design overview: cJQeyyhx9PraQrY3YAtCGM-fi-1104%3A152052

Screen Shot 2022-05-30 at 13 52 07

### How to test the changes in this Pull Request:

1. Run `pnpm install`
2. Run `pnpm nx build @woocommerce/components`
3. Run `pnpm storybook`
4. Go to http://localhost:6007/
5. Find Tour-kit component
6. Confirm it matches the design

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [x] Have you written new tests for your changes, as applicable?
-   [x] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm nx changelog <project>`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Comment by botwoo** [comment]
:bar_chart: Test reports for this pull request have been published and are accessible through the following links:

- [REST API Test Report](https://woocommerce.github.io/woocommerce-test-reports/pr/33229/api/)
- E2E Test Report (TBC)

Latest commit referenced in the reports: `Update type doc` f924765dc8b656b119e585d7005f70a84a27acb1
This comment will automatically be updated with the latest referenced commit when you push new changes to this pull request.

---

_Visit the [WooCommerce Test Reports](https://woocommerce.github.io/woocommerce-test-reports) homepage to view all published reports. See the [FAQs](https://github.com/woocommerce/woocommerce-test-reports/wiki/FAQs) page if you're having problems accessing them._

File: packages/js/components/src/tour-kit/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# tour-kit
```
Comment (adrianduffell) [inline packages/js/components/src/tour-kit/README.md:1]: Thanks for the nice README!

**Review by adrianduffell — APPROVED** [review]
This tested nice, and code looks great. Awesome work, thank you @chihsuan !

I have a few suggestions which could be considered in follow-up PRs.

*  Potentially the code could be moved to the [experimental package](https://github.com/woocommerce/woocommerce/tree/trunk/packages/js/experimental) if we want to iterate on it a bit before making it public.

* We could possibly bolster the test coverage with a few more tests, maybe around inputting text into fields, and verifying behavior on the last step.

**Comment by chihsuan** [comment]
> I have a few suggestions which could be considered in follow-up PRs.
> Potentially the code could be moved to the [experimental package](https://github.com/woocommerce/woocommerce/tree/trunk/packages/js/experimental) if we want to iterate on it a bit before making it public.

> We could possibly bolster the test coverage with a few more tests, maybe around inputting text into fields, and verifying behavior on the last step.

Thanks. Sounds good!

**Pull request merged**